source/features/show-stargazers-you-know.js

```diff
@@ -0,0 +1,75 @@
+import {h} from 'dom-chef';
+import select from 'select-dom';
+import domify from '../libs/domify';
+import {getRepoURL} from '../libs/page-detect';
+
+const extractUserData = element => {
+	const image = element.querySelector('img');
+	const link = element.querySelector('a').href;
+	return {
+		avatar: image.src.replace(/\?.+/, ''),
```

Comment:
Better to use `new URL(image.src)` and properly pull out or remove what you want.